src/HomeKit/HMEnums.cs

```diff
@@ -1070,6 +1108,18 @@ public enum HMAccessoryCategoryType {
 		[iOS (10,2), TV (10,1)]
 		[Field ("HMAccessoryCategoryTypeAirDehumidifier")]
 		AirDehumidifier,
+
+		[Watch (4,2), TV (11,2), iOS (11,2)]
+		[Field ("HMAccessoryCategoryTypeSprinkler")]
+		Sprinkler,
+
+		[Watch (4,2), TV (11,2), iOS (11,2)]
+		[Field ("HMAccessoryCategoryTypeFaucet")]
+		Faucet,
+
+		[Watch (4,2), TV (11,2), iOS (11,2)]
+		[Field ("HMAccessoryCategoryTypeShowerHead")]
+		ShowerHead,
```

Comment:
Reeeeeeeeeeally HomeKit shower heads... Ahem no comment (:

Reply:
Ah, the joys of powered shower heads. There's a reason they're called [suicide showers](https://www.google.es/search?q=suicide+shower)...